articles/becoming-a-flutter-developer/index.md

```diff
@@ -0,0 +1,179 @@
+
+They test the functionality of the applications to make sure it does not crash. It also ensures the apps are running the well based on the system and user requirements.
+
+3. **To Deliver Applications.**
+
+It involves launching a complete application to the public. A mobile app is placed on `PlayStore` for Android applications or `AppleStore` for iOS applications. The feedback from the users is monitored and used to make changes and update the applications.
+
+
+### Flutter Developer Tools
+
+1. **Version Control Tools**
+
+These tools allow software developers to keep track of the changes they make on their projects. They also help for easy collaboration of a team of software developers while contributing to the same project. 
+
+Examples of version control tools:
+- [Git and Github](https://github.com/)
+- [Bitbucket](https://bitbucket.org/)
+
+2. **Flutter Framework**
+
+Flutter is a UI framework used to develop high performance and high-quality native applications for Android, iOs, the Web and desktop using a single code base. Dart is the primary programming language for Flutter Framework. It is a free and open-source framework. 
+
+Some features of Flutter that make it popular include 
+
+1. `Hot Reloading` which enables developers to see changes from the codebase in the UI. 
+2. `Hot Restart`. 
+3. `Present UI element` to create beautiful app designs for better `UI and UX`. 
+
+These UI elements are 
+1. Cupertino 
+2. Material Design. 
+
+These built-in features make building Flutter applications fast in performance. 
+
+
+3. **Integrated Development Environment**
+
+In using Flutter, you will choose between three IDEs. Which are:
+- Android Studio.
+- Visual Studio Code.
+- IntelliJ IDEA.
+
+Most Flutter developers chose between Android Studio and Visual Studio Code. They chose IDEs depending on the features suitable to them. Android Studio and Visual Studio Code are both capable of creating full Flutter apps, developer preferences aside.
+
+4. **DevTools**
+
+It is a tooling web suite for Flutter and Dart developers comprising a layout inspection tools, performance tools and memory tools. It contains all the debugging tools that you need to be an efficient and effective Flutter developer. 
+
+5. **Design Tools**
+
+These are tools used to craft and visualize ideas. Design is an important stage of app development. It ensures for a smooth software development process. Some software development teams include UI/UX designers, especially for building commercial applications while others lack. Building personal projects will also need knowledge of design tools.
+
+Here are some of the most popular design tools:
+- Sketch.
+- Adobe XD.
+- [Figma.](https://www.figma.com/) 
+- [Flutter Studio.](https://flutterstudio.app/)
+
+After designing, you can use [Supernova](https://www.supernova.io/) to convert the design into code by exporting it. You will, however, need to refactor the exported code to break it into readable code.
```

Comment:
Does supernova support all the listed designs (Figma, Adobe XD etc. Please clarify for the reader. You can also note that Sketch only works on macs.